caluma/form/schema.py

```diff
@@ -10,7 +10,14 @@
 
 
 class QuestionJexl(graphene.String):
-    """Question jexl represents a jexl expression returning boolean."""
+    """Question jexl expression returning boolean.
+
+    Following transform can be used:
+    * answer - access answer of document by question slug
+
+    Example:
```

Comment:
I feel all the JEXL stuff should be documented somewhere as well (ie. where can I use jexl, what transforms/data do I have available, etc).

Reply:
I have added a small section on the README.

Small note though: Docstrings like this will actually be automatically extracted into a GraphQL schema documentation which is referenced how to open it on the README already.